ext/xmlrpc/xmlrpc.stub.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+function xmlrpc_encode($value): string {}
```

Comment:
It seems the function may also return `NULL`, so:
```suggestion
function xmlrpc_encode($value): ?string {}
```